### PR TITLE
WIP: Camelk operator onboarding

### PR DESCRIPTION
**Pull request by macharya0907**

### CODE REVIEW

**Comment by adtandale** [comment]
Hi @macharya0907 , if you think tht 'work in progress' please add 'WIP'. Also, have you executed command 'make update' after you commit. If not, please do it so the automatic jobs will get executed.

**Comment by adtandale** [comment]
HI @macharya0907 , seems prow/owners job is failing as you didnt added OWNERS filr ci-operators/jobs/apache/camel-k location. Please add it.

**Comment by adtandale** [comment]
HI @macharya0907 , seems check-gh-automation job as 'openshift-ci' bit is not installed in the repo.

**Comment by macharya0907** [comment]
> HI @macharya0907 , seems prow/owners job is failing as you didnt added OWNERS filr ci-operators/jobs/apache/camel-k location. Please add it.

added it, thanks!

**Comment by macharya0907** [comment]
> Hi @macharya0907 , if you think tht 'work in progress' please add 'WIP'. Also, have you executed command 'make update' after you commit. If not, please do it so the automatic jobs will get executed.

done!

**Comment by adtandale** [comment]
Hi @macharya0907 , the job - 
ci/prow/prow-config-filenames  is failing due to an error- ERROR: Only .yaml extensions are allowed, not .yml as in apache/camel-k/apache-camel-k-main__camelk-ocp4.13-lp-interop.yml , please change the extension of file.

**Comment by macharya0907** [comment]
> Hi @macharya0907 , the job - ci/prow/prow-config-filenames is failing due to an error- ERROR: Only .yaml extensions are allowed, not .yml as in apache/camel-k/apache-camel-k-main__camelk-ocp4.13-lp-interop.yml , please change the extension of file.

I see the file **apache-camel-k-main__camelk-ocp4.13-lp-interop.yaml** already exist in config folder.

**Comment by adtandale** [comment]
@macharya0907 you need to change file format to .yaml no yml for apache/camel-k/apache-camel-k-main__camelk-ocp4.13-lp-interop.yml

**Comment by adtandale** [comment]
@macharya0907 I am seeing the error - level=error msg=Cluster initialization failed because one or more operators are not functioning properly. YOu can make change and run the job again.

File: ci-operator/config/apache/camel-k/apache-camel-k-main__camelk-ocp4.13-lp-interop.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: camelk-amp-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
```
Comment (chetna14manku) [inline ci-operator/config/apache/camel-k/apache-camel-k-main__camelk-ocp4.13-lp-interop.yaml:30]: please update BASE_DOMAIN to `cspilp.interop.ccitredhat.com` that will resolve another error you are getting:

> failed to fetch Common Manifests: failed to fetch dependency of "Common Manifests": failed to generate asset "DNS Config": getting public zone for "aws.interop.ccitredhat.com": No public route53 zone found matching name "aws.interop.ccitredhat.com"

File: ci-operator/config/apache/camel-k/apache-camel-k-main__camelk-ocp4.13-lp-interop.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      OPERATORS: |
+        [
+            {"name": "camel-k-operator", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators", "target_namespaces": "","operator_group": "global-operators"}
```
[the remark this answers is not in the document]
Comment (macharya0907) [reply ci-operator/config/apache/camel-k/apache-camel-k-main__camelk-ocp4.13-lp-interop.yaml]: changed it.

File: ci-operator/config/apache/camel-k/apache-camel-k-main__camelk-ocp4.13-lp-interop.yaml (hunk 1 above)
Comment (macharya0907) [reply ci-operator/config/apache/camel-k/apache-camel-k-main__camelk-ocp4.13-lp-interop.yaml]: changed

**Pull request closed without merging**